reward/rewardConfigCache.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package reward
+
+import (
+	"errors"
+	"github.com/hashicorp/golang-lru"
+	"github.com/klaytn/klaytn/params"
+	"math/big"
+	"strconv"
+	"strings"
+)
+
+var (
+	FailGettingConfigure = errors.New("fail to get configure from governance")
+	InvalidFormat        = errors.New("invalid format")
+	ParsingError         = errors.New("parsing error")
+)
+
+const (
+	maxRewardConfigCache = 5 // Must be positive integer
```

Comment:
How about removing the comment '// Must be positive integer'? I think that is obvious.

Reply:
reflected